### DISCUSSION TITLE
v.5.1.1 New Install, slow webUI

### DISCUSSION THREAD

**Discussion by UberPlexCa** [openbroadcaster/obplayer#60] [Q&A]
Hi All,

Brand new install on Xununtu,  it all starts up fine, but the webui is slow and times out when saving or changing tabs.   Anyone else seeing this?

Watching the realtime logs and if I press the stop button on the browser it cuts the connection and then I can do something else but then it freezes again.

**Comment by wmcgrathled** [comment]
I have a similar problem on my new Raspberry PI install.  obplayer uses 120% of CPU and is very very slow.  

I will see if I can capture the problem with a log. 

Apparently when I connect to the player site (:23233), it throws some kind of exception and goes from 25% to 100%-130% CPU usage.
Sumpin ain't right.  I really want to get this working as I love some of the new features!

**Reply by UberPlexCa** [reply]
I just did a new install on Ubuntu with the same issues so I opened a bug report   https://github.com/openbroadcaster/obplayer/issues/81
